erpnext/hr/report/monthly_attendance_sheet/monthly_attendance_sheet.js

```diff
@@ -5,34 +5,19 @@
 frappe.query_reports["Monthly Attendance Sheet"] = {
 	"filters": [
 		{
-			"fieldname": "month",
-			"label": __("Month"),
-			"fieldtype": "Select",
-			"reqd": 1 ,
-			"options": [
-				{ "value": 1, "label": __("Jan") },
-				{ "value": 2, "label": __("Feb") },
-				{ "value": 3, "label": __("Mar") },
-				{ "value": 4, "label": __("Apr") },
-				{ "value": 5, "label": __("May") },
-				{ "value": 6, "label": __("June") },
-				{ "value": 7, "label": __("July") },
-				{ "value": 8, "label": __("Aug") },
-				{ "value": 9, "label": __("Sep") },
-				{ "value": 10, "label": __("Oct") },
-				{ "value": 11, "label": __("Nov") },
-				{ "value": 12, "label": __("Dec") },
-			],
-			"default": frappe.datetime.str_to_obj(frappe.datetime.get_today()).getMonth() + 1
+			"fieldname": "from_date",
+			"label": __("From date"),
+			"fieldtype": "Date",
+			"reqd": 1
 		},
 		{
-			"fieldname":"year",
-			"label": __("Year"),
-			"fieldtype": "Select",
+			"fieldname": "to_date",
+			"label": __("To date"),
```

Comment:
```suggestion
			"label": __("To Date"),
```
Labels in title case